sdk/nodejs/apps/v1/StatefulSet.ts

```diff
@@ -118,7 +118,6 @@ import { getVersion } from "../../version";
                   { parent: opts.parent, type: "kubernetes:apps/v1:StatefulSet", name: name },
                   { parent: opts.parent, type: "kubernetes:apps/v1beta1:StatefulSet", name: name },
                   { parent: opts.parent, type: "kubernetes:apps/v1beta2:StatefulSet", name: name },
-                  { parent: opts.parent, type: "kubernetes:extensions/v1beta1:StatefulSet", name: name },
```

Comment:
Was this just a bug in our previous logic?

Reply:
Yes, it appears so. It looks like the [deprecation announcement](https://kubernetes.io/blog/2019/07/18/api-deprecations-in-1-16/) I referred to previously was incorrect.